server/authz/rbac.go

```diff
@@ -153,3 +175,82 @@ func extractObject(c *gin.Context) string {
 	}
 	return emptyString
 }
+
+// getRbacProperty is used to read the RbacPropertiesPath file path and extract the policy provided as argument,
+func getRbacProperty(property string, config *viper.Viper) interface{} {
+	val := config.Get(property)
+	if val == nil {
+		return emptyString
+	}
+	return val
+}
+
+// parseProperty parses the property from the rbac properties file.
+// The property is in the format "property-name: value"
+func parseProperty(line string) (string, string) {
+	// line is in the format "property-name: value"
+	branchArray := strings.SplitN(line, ":", 2)
+	propertyName := strings.TrimSpace(branchArray[0])
+	value := strings.TrimSpace(branchArray[1])
+	return propertyName, value
+}
+
+// GetRbacScopes returns the scopes from the rbac properties file. If no scopes are provided, it returns Group as the
+// default scope. The scopes are used to determine the user identity token to be used for authorization.
+// If the scope is group, the user identity token will be the groups assigned to the user from the authentication
+// system. If the scope is email, the user identity token will be the email assigned to
+// the user from the authentication.
+// The scopes are provided as a comma separated list in the rbac properties file.
+// Example: policy.scopes=groups,email
+func GetRbacScopes(config *viper.Viper) []string {
```

Comment:
```suggestion
func GetRBACScopes(config *viper.Viper) []string {
```

[initialism](https://zchee.github.io/golang-wiki/CodeReviewComments/#initialisms)